frontends/p4/fromv1.0/converters.h

```diff
@@ -78,12 +78,23 @@ class TypeConverter : public ExpressionConverter {
     explicit TypeConverter(ProgramStructure* structure) : ExpressionConverter(structure) {}
 };
 
+class ExternConverter {
```

Comment:
Yes, that is what I meant.
Or it could be a pure virtual class, and the users have to provide some implementation.
Most policies in the mid-end are described by pure virtual classes. This one is not in the mid-end, though.